### PR TITLE
removed kafkacheckpoint migration script from dependecy list

### PR DESCRIPTION
**Pull request by Rohit25negi**

### CODE REVIEW

**Review by snopoke — COMMENTED** [review]
Build failure on sharded build looks related:
```
  Applying pillowtop.0001_initial... OK
  Applying pillowtop.0002_djangopillowcheckpoint_sequence_format... OK
  Applying pillowtop.0003_auto_20170411_1957... OK
  Applying cleanup.0001_convert_change_feed_checkpoint_to_sql... OK
  Applying cleanup.0002_convert_mc_checkpoint_to_sql... OK
  Applying cleanup.0003_convert_fluff_checkpoints_to_sql... OK
  Applying cleanup.0004_convert_ucr_checkpoints_to_sql... OK
  Applying cleanup.0005_convert_mvp_checkpoints_to_sql... OK
  Applying cleanup.0006_convert_report_es_checkpoints_to_sql... OK
  Applying cleanup.0007_convert_es_checkpoints_to_sql... OK
  Applying cleanup.0008_convert_sofabed_checkpoints_to_sql... OK
  Applying cleanup.0009_convert_final_checkpoints_to_sql... OK
  Applying cleanup.0010_rename_default_change_feed_checkpoint... OK
  Applying cleanup.0011_merge_couch_sql_pillows... OK
  Applying cleanup.0012_add_es_index_to_checkpoint_ids...Traceback (most recent call last):
  File "./manage.py", line 137, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python2.7/site-packages/django/core/management/commands/migrate.py", line 204, in handle
    fake_initial=fake_initial,
  File "/usr/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 115, in migrate
    state = self._migrate_all_forwards(state, plan, full_plan, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 145, in _migrate_all_forwards
    state = self.apply_migration(state, migration, fake=fake, fake_initial=fake_initial)
  File "/usr/local/lib/python2.7/site-packages/django/db/migrations/executor.py", line 244, in apply_migration
    state = migration.apply(state, schema_editor)
  File "/usr/local/lib/python2.7/site-packages/django/db/migrations/migration.py", line 129, in apply
    operation.database_forwards(self.app_label, schema_editor, old_state, project_state)
  File "/mnt/commcare-hq-ro/corehq/sql_db/operations.py", line 31, in database_forwards
    super(HqOpMixin, self).database_forwards(app_label, schema_editor, from_state, to_state)
  File "/usr/local/lib/python2.7/site-packages/django/db/migrations/operations/special.py", line 193, in database_forwards
    self.code(from_state.apps, schema_editor)
  File "/mnt/commcare-hq-ro/corehq/apps/cleanup/migrations/0012_add_es_index_to_checkpoint_ids.py", line 32, in copy_checkpoints
    [pillow_instance.checkpoint.checkpoint_id for pillow_instance in get_all_pillow_instances()]
  File "./corehq/ex-submodules/pillowtop/utils.py", line 40, in get_all_pillow_instances
    return [config.get_instance() for config in get_all_pillow_configs()]
  File "./corehq/ex-submodules/pillowtop/utils.py", line 88, in get_instance
    return instance_generator_fn(self.name, **params)
  File "/mnt/commcare-hq-ro/corehq/pillows/case.py", line 43, in get_case_to_elasticsearch_pillow
    checkpoint = get_checkpoint_for_elasticsearch_pillow(pillow_id, CASE_INDEX_INFO, topics.CASE_TOPICS)
  File "./corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 230, in get_checkpoint_for_elasticsearch_pillow
    return KafkaPillowCheckpoint(checkpoint_id, topics)
  File "./corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 171, in __init__
    self._get_checkpoints()
  File "./corehq/ex-submodules/pillowtop/checkpoints/manager.py", line 174, in _get_checkpoints
    return KafkaCheckpoint.get_or_create_for_checkpoint_id(self.checkpoint_id, self.topics)
  File "./corehq/ex-submodules/pillowtop/models.py", line 93, in get_or_create_for_checkpoint_id
    _create_checkpoints_from_kafka(checkpoint_id, topics)
  File "./corehq/ex-submodules/pillowtop/models.py", line 108, in _create_checkpoints_from_kafka
    defaults={"offset": 0}
  File "/usr/local/lib/python2.7/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 464, in get_or_create
    return self.get(**lookup), False
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 374, in get
    num = len(clone)
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 232, in __len__
    self._fetch_all()
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 1121, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "/usr/local/lib/python2.7/site-packages/django/db/models/query.py", line 53, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch)
  File "/usr/local/lib/python2.7/site-packages/django/db/models/sql/compiler.py", line 899, in execute_sql
    raise original_exception
django.db.utils.ProgrammingError: column pillowtop_kafkacheckpoint.doc_modification_time does not exist
LINE 1: ...partition", "pillowtop_kafkacheckpoint"."offset", "pillowtop...
```

File: corehq/ex-submodules/pillowtop/migrations/0003_auto_20170411_1957.py [hunk 1]
```diff
@@ -11,7 +11,9 @@ class Migration(migrations.Migration):
     dependencies = [
         ('pillowtop', '0002_djangopillowcheckpoint_sequence_format'),
     ]
-
+    run_before = [
+        ('pillowtop', '0004_offset_to_big_int'),
+    ]
```
Comment (snopoke) [inline corehq/ex-submodules/pillowtop/migrations/0003_auto_20170411_1957.py]: Why are these `run_before` necessary?

Comment (Rohit25negi) [reply corehq/ex-submodules/pillowtop/migrations/0003_auto_20170411_1957.py]: I was trying to fix this https://github.com/dimagi/commcare-hq/pull/21629#discussion_r222178272. Still not able to figure out why travis is not picking up the  `0005_kafkacheckpoint_doc_modification_time`.

**Comment by Rohit25negi** [comment]
Talked to @snopoke,  and understood that `0012_add_es_index_to_checkpoint_ids` was a one time migration and by now it should be run in all envs, so it can be empty now. which should resolve this problem.

**Pull request merged**